lib/src/google_fonts_base.dart

```diff
@@ -172,8 +172,10 @@ Future<void> loadFontIfNecessary(GoogleFontsDescriptor descriptor) async {
     _loadedFonts.remove(familyWithVariantString);
     print('Error: google_fonts was unable to load font $fontName because the '
         'following exception occured:\n$e');
-    if (file_io.isMacOS) {
-      print('See https://docs.flutter.dev/desktop#setting-up-entitlements.');
+    if (file_io.isMacOS || file_io.isAndroid) {
+      print(
+        'See https://docs.flutter.dev/development/data-and-backend/networking#platform-notes.',
```

Comment:
this does not appear to show anything about the MacOS entitlements, is that still needed? It is mentioned in docs above

Reply:
One step ahead :) https://github.com/flutter/website/pull/7169

Reply:
fantastic